app/views/declaration/items_summary.scala.html

```diff
@@ -30,79 +42,142 @@
     else { messages("supplementary.itemsAdd.titleWithItems", items.length) }
 }
 
-@main_template(
-    title = Title(titleMessage, "supplementary.items")
-) {
-    @components.back_link(routes.PreviousDocumentsController.displayPage(mode), mode)
+@pageHint = {
+    <span class="govuk-hint">
+        @messages("supplementary.itemsAdd.title.hint")
+    </span>
+}
 
-    @components.error_summary(itemsErrors)
+@addButtonMessage = @{
+    if(items.isEmpty) { messages("site.add.item") }
+    else { messages("site.add.anotherItem") }
+}
 
-    @components.heading(
-        sectionHeader = messages("supplementary.items"),
-        title = titleMessage,
-        hint = messages("supplementary.itemsAdd.title.hint")
-    )
+@addButton = {
+    @govukButton(Button(
```

Comment:
There is already an "addButton" component in components/gds that you can use.  You created it, lol

Reply:
Hum... that wasn't me. My name go assigned to it because of some Git rescue effort.
Also that `addButton` has a bug on the visuallyHidden message

Reply:
I have refactored the code and have reused it.